src/renderer/modules/components/ErrorBoundary.tsx

```diff
@@ -40,11 +86,19 @@ export default class ErrorBoundary extends React.Component<ErrorProps, ErrorStat
 
   public render(): React.ReactNode {
     if (this.state.hasError) {
+      const { error, pluginName } = this.state;
+
       return (
         this.props.fallback || (
           <div className="replugged-error-boundary">
             <h1>{Messages.REPLUGGED_SETTINGS_ERROR_HEADER}</h1>
+            {pluginName && (
+              <p className="replugged-error-boundary-plugin">
+                {Messages.REPLUGGED_SETTINGS_ERROR_PLUGIN_NAME.format({ name: pluginName })}
```

Comment:
Just to make sure this won't error
```suggestion
                {Messages.REPLUGGED_SETTINGS_ERROR_PLUGIN_NAME?.format?.({ name: pluginName })}
```